mars/lib/filesystem/tests/test_filesystem.py

```diff
@@ -198,3 +199,14 @@ def test_fsmap():
         # create root
         fs_map = FSMap(root + "/path2", fs, create=True)
         assert len(fs_map) == 0
+
+
+def test_get_fs():
```

Comment:
As `fsspec` is an extra dependency, use `pytest.mark.skipif` to run cases only when `fsspec` is installed.

Reply:
Good point. Will do it.